### PR TITLE
Include user pronouns in relevant commands. Fix NameError.

### PR DESCRIPTION
**Pull request by gotmax23**

### CODE REVIEW

**Comment by gotmax23** [comment]
I tested this with https://github.com/ryanlerch/maubot-plugindevenv/. I ran `vagrant up`, sshed in, set up a kerberos ticket for fasjson, authenticated with a different Matrix account I created, created an instance in the webui, and PMed the bot account.

File: fedora/__init__.py [hunk 1]
```diff
@@ -373,3 +377,13 @@ async def whoowns(self, evt: MessageEvent, package: str) -> None:
         resp = "".join([x for x in [owners, admins, committers] if x != ""])
 
         await evt.reply(resp)
+
+    @command.new(help="Retrive a user's pronouns")
+    @command.argument("username", pass_raw=True, required=False)
+    async def pronouns(self, evt: MessageEvent, username: str) -> None:
```
Comment (ryanlerch) [inline fedora/__init__.py:383]: hmmm, not sure we need a seperate command for this -- is returning them with the userinfo not enough?

File: fedora/__init__.py [hunk 2]
```diff
@@ -373,3 +377,13 @@ async def whoowns(self, evt: MessageEvent, package: str) -> None:
...
+        # XXX: Why does this sometimes return a string?
+        user = self._get_fasuser(username, evt)
+        if isinstance(user, str):
```
Comment (ryanlerch) [inline fedora/__init__.py:386]: _get_user() returns a string if it hits an issue, and the string is what we have to respond to the user. there is probably a better way of doing this TBH

File: fedora/__init__.py (hunk 1 above)
Comment (gotmax23) [reply fedora/__init__.py:383]: I like having a simple, one line output command for this, and I don't think it adds too much of a maintenance version

File: fedora/__init__.py (hunk 2 above)
Comment (gotmax23) [reply fedora/__init__.py:386]: Ah, okay. I'll just have it return the error message directly.

File: fedora/__init__.py (hunk 1 above)
Comment (ryanlerch) [reply fedora/__init__.py:383]: i just worry about command bloat really -- dont want to be adding commands for each field in a user's Fedora Account.

**Comment by ryanlerch** [comment]
i picked the other three commits, and pushed them (and they are now running on the stage zodbot)

**Pull request closed without merging**